UnitySDK/Assets/ML-Agents/Editor/DemonstrationImporter.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using UnityEngine;
+using UnityEditor.Experimental.AssetImporters;
+
+namespace MLAgents
+{
+    /// <summary>
+    /// Asset Importer used to parse demonstration files.
+    /// </summary>
+    [ScriptedImporter(1, new[] {"demo"})]
```

Comment:
"demo" is very general. Does it make sense to have a more unique extension?

Reply:
Any suggestions? `.demon`? 👿